spec/fortnox/api/types/account_number_spec.rb

```diff
@@ -13,19 +13,17 @@
     it { is_expected.to be_nil }
   end
 
-  context 'when AccountNumber created with empty string' do
```

Comment:
@samuel02 Why dropping this case? Is it not needed anymore?

Reply:
It got complicated :) The problem is that the constructor `Dry::Types::Coercions::Form.to_int` will return `nil` in the case of an empty string. And since the type is declared as optional that's ok and it won't raise anything. What do you think? Should I write a custom constructor that handles that case as well?

Reply:
Well, it being complicated is hardly a reason to not test for it. In fact it is a very good reason TO test for it! 😄

Reply:
Sure :) My thinking was just that it doesn't maybe need to be *that* strict. The consequence of the new code would be that an empty string would end up as a nil value which was already permitted. So in practice it probably doesn't matter that much. But sure :) I pushed another implementation. Unfortunately it's not enough with the constructor you provided since `Integer` doesn't implement `#empty?`, if we'd have `ActiveSupport` we could have used `Object#blank?` but I didn't want to pull that in. So I ended up writing my own. WDYT?